katalog/cert-manager/webhook/deploy.yml

```diff
@@ -23,7 +23,7 @@ spec:
         imagePullPolicy: Always
         args:
         - --v=2
-        - --secure-port=6443
+        - --secure-port=10250
```

Comment:
Why 10250? is it a magic number? I mean... it remembers me the default kubelet port (https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/)
Is it intended?

Reply:
@angelbarrera92 It is just that port `10250` is an already exposed port by GKE. Like 443.